tests/Integration/Lib/User/IntegrationTestUserAvatar.php

```diff
@@ -65,8 +65,20 @@ private function execFetchTest($dn, $username, $image) {
 		// initialize home folder and make sure that the user will update
 		// also remove an possibly existing avatar
 		\OC_Util::tearDownFS();
+
+		\OC_User::setUserId($username);
+		$userObject = \OC::$server->getUserManager()->get($username);
+		if (!is_null($userObject)) {
+		    $userObject->updateLastLoginTimestamp();
+		}
 		\OC_Util::setupFS($username);
-		\OC::$server->getUserFolder($username);
+		if (\OC_User::userExists($username)) {
```

Comment:
This will likely check the existence against the account table, but the user might not be there yet.